src/Domain/Model/AssociablePayment.php

```diff
@@ -0,0 +1,15 @@
+<?php
+declare( strict_types=1 );
+
+namespace WMDE\Fundraising\PaymentContext\Domain\Model;
+
+/**
+ * @template T of Payment
+ */
+interface AssociablePayment {
+	/**
+	 * @param int $followUpPaymentId
+	 * @return T
+	 */
+	public function createFollowUpPayment( int $followUpPaymentId ): Payment;
+}
```

Comment:
Somehow this interface snuck back in, probably through the rebase. Please remove it